### PR TITLE
Added Content Script for Select Button

### PR DESCRIPTION
**Pull request by varunsivashankar**
**OUSelectContent.js**
- In this file, there is a function _selectClick_ that updates the names of preferred entities in the event that some names have been changed.
- This function gets the data row IDs of preferred entities from the DOM 
- It then checks whether each of these entities is present in chrome storage, and if so, whether that particular entity has a new name.
- TODO: Tests for this function need to be added.

**popup.js**
- This code described above is executed when the select button of the extension is clicked.
- This is carried out by injecting the content script into the DOM when the select button is clicked.

### CODE REVIEW

File: STEP_Project/OUSelectContent.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+            }
+        }
+    });
+}
+
+(function()
+{
+    selectClick();
+}()
+);
```
Comment (atishey-jain) [inline STEP_Project/OUSelectContent.js:32]: nit: Add a new line at the end of file.

File: STEP_Project/popup.js [hunk 2]
```diff
@@ -12,6 +12,8 @@ const PAGE_TYPES = {
   checkTabUrl();
   addRemoveButton = document.getElementById("addRemove");
   injectContent(addRemoveButton);
+  let selectButton = document.getElementById("select");
+  injectSelectContent(selectButton);
```
Comment (atishey-jain) [inline STEP_Project/popup.js:16]: We are not actually injecting the select button here, right ? We are just adding event listener to an already existing button. Maybe we should rename this to something appropriate.

Comment (varunsivashankar) [reply STEP_Project/popup.js]: Renamed it to selectListener

File: STEP_Project/OUSelectContent.js (hunk 1 above)
Comment (varunsivashankar) [reply STEP_Project/OUSelectContent.js]: Done

**Pull request merged**